gateway/src/commonMain/kotlin/builder/PresenceBuilder.kt

```diff
@@ -36,6 +36,10 @@ public class PresenceBuilder {
         game = DiscordBotActivity(name, ActivityType.Competing)
     }
 
+    public fun custom(name: String, state: String) {
+        game = DiscordBotActivity(name, state = Optional(state), type = ActivityType.Custom)
```

Comment:
if i understand [this comment](https://github.com/discord/discord-api-docs/pull/6345#issuecomment-1672271748) correctly, this should look like this:
```suggestion
    public fun custom(state: String) {
        game = DiscordBotActivity(name = "Custom Status", state = Optional(state), type = ActivityType.Custom)
    }
```
or do you see `name` mentioned in the client somewhere?